module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -63,7 +70,7 @@ public function holdAction()
         $checkHolds = $catalog->checkFunction(
             'Holds',
             [
-                'id' => $driver->getUniqueID(),
+                'id' => $id,
```

Comment:
You could replace this whole array with `compact('id', 'patron')` now that the variable name and key names match up.